packages/functions/src/common/BookAppointmentHelper.ts

```diff
@@ -10,21 +10,31 @@ import {
   BookingChange,
   FunctionsApi,
   Hospital,
+  MANUAL_DONOR_ID,
 } from "@zm-blood-components/common";
 import { dbAppointmentToBookedAppointmentApiEntry } from "../utils/ApiEntriesConversionUtils";
 import { notifyOnAppointmentBooked } from "../notifications/BookAppointmentNotifier";
-import { BookAppointmentStatus } from "../../../common/src/functions-api";
+import { BookAppointmentStatus } from "@zm-blood-components/common/src/functions-api";
 import { DbDonor } from "../function-types";
 
 const WEEKS_BUFFER = 0;
 
-export default async function (
+export async function BookAppointment(
   request: FunctionsApi.BookAppointmentRequest,
+  isDonor: boolean,
   callerId: string
 ): Promise<FunctionsApi.BookAppointmentResponse> {
-  const donorId = callerId;
+  let donorId = callerId;
```

Comment:
This is confusing. 
Change the parameter name in line 25 to `donorId` and let the callers take care of this logic.
You can add another optional field called `bookingCoordinatorId` that will only apply to the new flow

Reply:
This way you can also remove the `isDonor` and deduce the caller of this function by the existence of the `bookingCoordinatorId`

Reply:
done